mysql-test/t/dd_upgrade_compression_dict-master.opt

```diff
@@ -0,0 +1 @@
+--bootstrap --innodb_page_size=16k
```

Comment:
No,  have_innodb_16k/4k/8k etc are removed. If not, they should be removed in our branch too.

--bootstrap initializes with the required page size.

The requirement comes from 5.7 datadir which is 16k. So we need to have running server which is 16k (the page size is set in mtr restart options somewhere).

Only 16k can be tested for upgrade now. Same with other upgrade tests

Reply:
https://github.com/mysql/mysql-server/blob/8.0/mysql-test/include/have_innodb_16k.inc present in upstream, since you know the circumstances better, can you please file a bug report with them?

Reply:
Ok, will check the tests using this .inc file and file a bug

Reply:
Reported upstream bug https://bugs.mysql.com/bug.php?id=93304